API_CHANGES.md

```diff
@@ -4,6 +4,100 @@ API Changes
 When an addition to the existing API is made, the minor version is bumped.
 When an API feature or function is removed or changed, the major version is bumped.
 
+2.25.0
+======
+Pointer class now supports `get_raw_value()`.
+`KTIMER` no longer supports `get_raw_dpc()`.
+
+2.24.0
+======
+Support `encoding` parameter for `objects.utility.array_to_string`
+
+2.23.0
+======
+Add support for windows GUI classes and OS distinguishers.
+Add a symbol_table_name for `ExecutiveObject.get_object_header()`/
+
+2.22.0
+======
+Linux net constants added.
+Network objects moved to separate versionable module.
+
+2.21.0
+======
+`uuid` method added to `linux.extensions`.
+
+2.20.0
+======
+NM_TYPES_DESC constants added to linux.
+`latch_tree_root` and `kernel_symbol` added to linux extensions.
+Linux `module` class additions:
+* `get_module_address_boundaries`
+* `section_typetab`
+Linux `task_struct` class additions:
+* `get_address_space_layer`
+* `state`
+Linux `bpf_prog` class additions:
+* `bpf_jit_binary_hdr_address`
+
+2.19.0
+======
+Introduction of `Modules` versionable linux extension module.
+Deprecation of some `LinuxUtilities` functions relating to modules.
+
+2.18.0
+======
+Addition of `scatterlist` linux extension.
+
+2.17.0
+======
+The addition of a `types` member to `SymbolInterface`
+
+2.16.0
+======
+Addition of TAINT_FLAG constants, `TaintFlag` dataclass
+Addition of linux `tainting` versionable module
+
+2.15.0
+======
+Addition of `convert_fourcc_code` to `LinuxUtilities` class
+
+2.14.0
+======
+No significant changes (part of the 2.16.0 PR which took time in development)
+
+2.13.0
+======
+Linux `task` objectr extension addition of `getppid`
```

Comment:
Just a tiny typo on object I think?  `Linux `task` object extension addition of `getppid``